src/main/java/eu/donals/Loader.java

```diff
@@ -1,9 +1,13 @@
+package eu.donals;
+
 import org.json.simple.JSONArray;
 import org.json.simple.JSONObject;
 
-public class LocationLoader {
+import java.util.List;
+
+public class Loader {
 
-    public static Location parseJsonObject(JSONObject location) {
+    private static Location parseLocationObject(JSONObject location) {
```

Comment:
Now that this is private, there is a Singleton property. Hope you have used that  for the design patterns!